relay-general/src/protocol/contexts/response.rs

```diff
@@ -0,0 +1,42 @@
+use crate::protocol::{Cookies, Headers};
+use crate::types::{Annotated, Object, Value};
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct ResponseContext {
+    /// Type `response`.
+    #[metastructure(field = "type")]
+    pub ty: Annotated<String>,
+
+    /// The cookie values.
+    ///
+    /// Can be given unparsed as string, as dictionary, or as a list of tuples.
+    #[metastructure(pii = "true", bag_size = "medium")]
+    #[metastructure(skip_serialization = "empty")]
+    pub cookies: Annotated<Cookies>,
+
+    /// A dictionary of submitted headers.
+    ///
+    /// If a header appears multiple times it, needs to be merged according to the HTTP standard
+    /// for header merging. Header names are treated case-insensitively by Sentry.
+    #[metastructure(pii = "true", bag_size = "large")]
+    #[metastructure(skip_serialization = "empty")]
+    pub headers: Annotated<Headers>,
+
+    /// HTTP status code.
+    pub status_code: Annotated<u64>,
+
+    /// HTTP response body size.
+    pub body_size: Annotated<u64>,
+
+    /// Additional arbitrary fields for forwards compatibility.
```

Comment:
Just adding some context here:

```suggestion
    /// Additional arbitrary fields for forwards compatibility.
    /// These fields are retained (`retain = "true"`) to keep supporting the format that the Dio integration sends:
    /// https://github.com/getsentry/sentry-dart/blob/7011abe27ac69bd160bdc6ecf3314974b8340b97/dart/lib/src/protocol/sentry_response.dart#L4-L8
```